### ISSUE TITLE
[Win8,Full-Screen] Outstanding issues in 32-bit mode

### ISSUE THREAD

**Issue by tomcw** [AppleWin/AppleWin#211]
Full screen **32-bit** issues: (Win8.1, VAIO)
- RHS icons, caps & drive only visible for a split second before disappearing
  - But icons & caps are fine in Start-up screen mode
  - Also switching to debugger (even from emulation mode), the icon & caps are fine
  - When pause icons are fine too.
- Dialogs don't erase the display (screen gets re-drawn correctly)
  - Both emulated screen & debugger

NB. For Win8.1 in Full screen **8-bit mode**: icon & CAPS are fine in debugger & when paused.
- Icons only visible briefly in 8-bit mode when emulating & not paused.
- Basically same behaviour as 32-bit more
